src/CommunityToolkit.Maui.UnitTests/Mocks/AnimationExtensions.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Maui;
+using Microsoft.Maui.Animations;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Handlers;
+
+namespace CommunityToolkit.Maui.UnitTests.Mocks;
+
+static class AnimationExtensions
+{
+	public static void EnableAnimations(this IView view) => MockAnimationHandler.Prepare(view);
+
+	// Inspired by Microsoft.Maui.Controls.Core.UnitTests.AnimationReadyHandler
+	class MockAnimationHandler : ViewHandler<IView, object>
```

Comment:
I've been jumping back on forth on whether we should try to make our naming match the MAUI naming to potentially make it easier to migrate features across should we get the approval. 

I think I am now leaning towards not doing that given we aren't guaranteeing that our underlying mocks match the exact behaviour of the MAUI mocks.

I am not entirely sure why I am writing these thoughts down now but what do you think?

And for the record I like the inclusion of Mock to make it clear what it does

Reply:
> whether we should try to make our naming match the MAUI naming to potentially make it easier to migrate features across should we get the approval

For `CommunityToolkit.Maui`, I 100% agree.

For `CommunityToolkit.Maui.UnitTests`, I don't 100% disagree, but I think what we have here is better and easier for other community members to leverage when they add new unit tests in the future.

>  I like the inclusion of Mock to make it clear what it does
Haha - thanks! I know I'm biased, but I think I improved on how the MAUI team is using `Microsoft.Maui.Controls.Core.UnitTests.AnimationReadyHandler` to enable animations.

I really like how clean `myLabel.EnableAnimations()` is, compared to their implementation:
```cs
var myLabel = AnimationReadyHandler.Prepare(new Label());
```

This extension method should make it super easy to enable animations for the unit tests in the future 💯

Reply:
Yes I agree this does feel like an improvement 😃 

I do like the extension method addition too. I have been experimenting with making mocks more descriptive by using creator/builder patterns such as:

```csharp
// creator - probably can find a better example
var myLabel = LabelCreator.WithAnimationsEnabled();

// builder
var myLabel = LabelBuilder
    .WithAnimationsEnabled()
    .WithBackgroundColor(Color.Red)
    .Build();
```

I do think what we have is descriptive though and easy to implement which is a really great thing!